### ISSUE TITLE
Uncaught Cannot run python command: check that python is available in the system (path: /usr/bin:/bin:/usr/sbin:/sbin): Error: spawn python ENOENT

### ISSUE THREAD

**Issue by reods** [sadovnychyi/autocomplete-python#11]
[Enter steps to reproduce below:]

1. ...
2. ...

**Atom Version**: 0.199.0
**System**: Mac OS X 10.9.5
**Thrown From**: [autocomplete-python](https://github.com/sadovnychyi/autocomplete-python) package, v0.3.3

### Stack Trace

Uncaught Cannot run python command: check that python is available in the system (path: /usr/bin:/bin:/usr/sbin:/sbin): Error: spawn python ENOENT

```
At /Users/edwardsr/.atom/packages/autocomplete-python/lib/provider.coffee:18

undefined
```

### Commands

```
     -1:17.7.0 window:install-shell-commands (atom-text-editor.editor.is-focused)
     -0:29.3.0 core:backspace (atom-text-editor.editor.mini.is-focused)
     -0:23 core:confirm (atom-text-editor.editor.mini.is-focused)
```

### Config

```json
{
  "core": {
    "themes": [
      "one-light-ui",
      "atom-dark-syntax"
    ]
  }
}
```

### Installed Packages

```coffee
# User
autocomplete-python, v0.3.3

# Dev
No dev packages
```

**Comment by sadovnychyi** [comment]
Where do you have your python installed? Can you give me the output of `which python` please?

**Comment by reods** [comment]
/usr/local/bin/python

**Comment by sadovnychyi** [comment]
I just released v0.4.0 with better path handling: in addition to system path plugin will try to locate python in some possible locations: `'/usr/local/bin', '/usr/bin', '/bin', '/usr/sbin', '/sbin'`

Also, if its still does not works for you -- you can set it manually in the package settings:
![image](https://cloud.githubusercontent.com/assets/193864/7719539/71167ea2-fef4-11e4-953b-ad37cf860252.png)

I'm closing this issue as resolved, but if you are still experiencing any problems -- feel free to submit a new one.
